enterprise/server/remote_execution/containers/podman/BUILD

```diff
@@ -53,3 +53,39 @@ go_test(
         "@org_golang_x_sync//errgroup",
     ],
 )
+
+# Repackages the podman-static distribution to make it more easily consumable
+# both by the container_layer rule and by tests:
+#
+# - Replaces the registry config with our custom one. This way, we don't have to
+#   override it separately.
+# - Strips the top-level podman-static directory. This is needed because the
+#   'tars' attribute of container_layer does not seem to support stripping
+#   components, currently.
+genrule(
+    name = "podman-static",
+    srcs = select({
+        "@platforms//cpu:x86_64": ["@com_github_buildbuddy_io_podman_static_podman-linux-amd64//file"],
+        "@platforms//cpu:aarch64": ["@com_github_buildbuddy_io_podman_static_podman-linux-arm64//file"],
+    }) + [
+        "//dockerfiles/executor_image:data/registries.conf",
+    ],
+    outs = ["podman-static.tar.gz"],
+    cmd_bash = """
+        TMPDIR=$$(mktemp -d)
+        trap 'rm -rf "$$TMPDIR"' EXIT
+        SRCS=($(SRCS))
+        PODMAN_ARCHIVE=$${SRCS[0]}
+        REGISTRIES_CONF=$${SRCS[1]}
+        tar --extract --file="$$PODMAN_ARCHIVE" --strip-components=1 --directory="$$TMPDIR"
+
+        # Remove README.md (not needed)
+        rm "$$TMPDIR/README.md"
```

Comment:
Nit:

```suggestion
        # Remove README.md (not needed)
        rm -f "$$TMPDIR/README.md"
```

Reply:
Same deal here, if the README no longer exists then we would want this to fail so that we could clean this up.